test/Libraries/RevitTestServices/RevitTestPathResolver.cs

```diff
@@ -6,6 +6,12 @@ namespace RevitTestServices
 {
     public class RevitTestPathResolver : TestPathResolver
     {
+        public RevitTestPathResolver() { }
```

Comment:
There still has one place use the constructor 'RevitTestPathResolver' without a parameter, could you check if we can replace it with our new constructor and remove the constructor without a parameter?

Reply:
It is not necessary to set this parameter in another place https://github.com/DynamoDS/DynamoRevit/blob/04ae617597d553a7e774eefad479a0060ea57f91/test/Libraries/RevitIntegrationTests/RegressionTests.cs#L48 . This is a DynamoRevit Node test, and I think it has no need to load some other local nodes.